libcst/helpers/__init__.py

```diff
@@ -4,3 +4,9 @@
 # LICENSE file in the root directory of this source tree.
 #
 # pyre-strict
+
+from libcst.helpers.expression import ensure_type, get_full_name_for_node
```

Comment:
ensure_type is not really an expression helper. It should probably go in a new module `libcst.helpers.common`.

Reply:
Moved `ensure_type` to `helpers.common`.